Shared/Samples/Trace utility network/TraceUtilityNetworkView.swift

```diff
@@ -0,0 +1,541 @@
+                }
+                MapViewReader { mapViewProxy in
+                    MapView(map: map, viewpoint: .initialViewpoint, graphicsOverlays: [points])
+                        .onSingleTapGesture { screenPoint, mapPoint in
+                            lastSingleTap = (screenPoint, mapPoint)
+                        }
+                        .selectionColor(.yellow)
+                        .confirmationDialog(
+                            "Select trace type",
+                            isPresented: $traceTypeSelectionIsOpen,
+                            titleVisibility: .visible,
+                            actions: { traceTypePickerButtons }
+                        )
+                        .confirmationDialog(
+                            "Select terminal",
+                            isPresented: $terminalSelectionIsOpen,
+                            titleVisibility: .visible,
+                            actions: { terminalPickerButtons }
+                        )
+                        .onChange(of: traceTypeSelectionIsOpen) { _ in
+                            // If type selection is closed and a new trace wasn't initialized we can
+                            // figure that the user opted to cancel.
+                            if !traceTypeSelectionIsOpen && pendingTraceParameters == nil { reset() }
+                        }
+                        .onDisappear {
+                            ArcGISEnvironment.authenticationManager.arcGISCredentialStore.removeAll()
+                        }
+                        .task {
+                            await setup()
+                        }
+                        .task(id: lastSingleTap?.mapPoint) {
+                            guard case .settingPoints = tracingActivity, let lastSingleTap else {
+                                return
+                            }
+                            if let feature = await identifyFeatureAt(
+                                lastSingleTap.screenPoint,
+                                with: mapViewProxy
+                            ) {
+                                add(feature, at: lastSingleTap.mapPoint)
+                            }
+                        }
+                        .task(id: tracingActivity) {
+                            updateUserHint()
+                            if tracingActivity == .tracing {
+                                await trace()
+                            }
+                        }
+                }
+                traceManager
+                    .frame(width: geometryProxy.size.width)
+                    .background(.thinMaterial)
+            }
+        }
+    }
+    
+    /// The view at the bottom of the screen that guides the user through the various stages of a
+    /// tracing activity.
+    var traceManager: some View {
+        HStack(spacing: 5) {
+            switch tracingActivity {
+            case .none:
+                Button("Start a new trace") {
```

Comment:
```suggestion
                Button("Start a New Trace") {
```
We use title case in the sample viewer for most situations

- https://developer.apple.com/design/human-interface-guidelines/foundations/writing/
- https://developer.apple.com/library/archive/documentation/FinalCutProX/Conceptual/FxPlugHIG/TextStyleGuidelines/TextStyleGuidelines.html